robotd/game_specific.py

```diff
@@ -0,0 +1,18 @@
+"""Game specific values."""
```

Comment:
I think this file has been committed accidentally. This file was moved to `sb-vision`

Reply:
+1

Reply:
Huh, this file was there when I started the branch, though it did get removed by a merge. Annoying that it's come back.

Reply:
01440da